### ISSUE TITLE
Node-gyp rebuild fails on Node.JS 0.12.0 when installing node-ncurses

### ISSUE THREAD

**Issue by pmcmonagle** [mscdex/node-ncurses#55]
Using Node.JS version 0.12.0, `npm install ncurses` results in a long string of errors:
```
../src/binding.cc:84:1: error: unknown type name 'uv_poll_t'
../src/binding.cc:667:34: error: unknown type name 'uv_handle_t'
../src/binding.cc:688:37: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
../src/binding.cc:721:39: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?
../src/binding.cc:731:38: error: unknown type name 'Arguments'; did you mean 'v8::internal::Arguments'?

etc.
```

Rolling back to Node.JS 0.10.37 (eg. via NVM) solves the problem.

Using:
- Mac OSX 10.10.3
- Node.JS   0.12.0
- NPM         2.5.1

**Comment by mscdex** [comment]
This module hasn't been updated to use `nan` yet and I don't have the time to do this. There was a PR to add support for this, but `nan` has changed quite a bit since then and would need a new PR, which I am more than willing to accept.

**Comment by mscdex** [comment]
Should be fixed with 507638965a.
